### PR TITLE
Cluster diagnostics

### PR DESCRIPTION
**Pull request by Arau**
Add cluster diagnostics in the support page. 

More information about the cluster diagnostics can be added in the future as this is a first step.

### CODE REVIEW

File: _docs/introduction/support.md [hunk 1]
```diff
@@ -44,3 +47,20 @@ entries, that may help us debug your issue.
 - Specific for your namespaces: `kubectl describe pvc PVC_NAME` 
 - Specific for your namespaces: `kubectl describe pod POD_NAME` 
 
+## StorageOS Cluster Report
+
+StorageOS has a `cluster report` functionality that aggregates logs, hardware
+information, cluster information and metadata. That bundle is created only
+under a user action. The bundle is cyphered and then uploaded after a user
```
Comment (actionbuddha) [inline _docs/introduction/support.md]: 'encrypted' - how? Details are useful.

**Comment by actionbuddha** [comment]
At the moment, it isn't clear from the docs why or when they should use the info bundler. We should make clear that StorageOS engineers may request users to run this during a support engagement.

**Comment by Arau** [comment]
I added corrections as suggested

File: _docs/introduction/support.md (hunk 1 above)
Comment (Arau) [reply _docs/introduction/support.md]: Questions answered.

**Pull request merged**